File: Modul 7/NO 3/PRAK603-2210817310016-Riyo Aurora Guison.c
#include <stdio.h>
int main()
{
    int a,b,i,hasil;
    scanf("%d %d",&a,&b);
    if(a==b){
    int baris1[a];int baris2[b];
    for(i=0;i<a;i++){
        scanf("%d",&baris1[i]);
    }
    for(i=0;i<b;i++){
        scanf("%d",&baris2[i]);
    }
    for(i=0;i<a;i++){
        hasil=baris1[i]*baris2[i];
        printf("%d ",hasil);
    }
    }
    else{
        printf("Jumlah tidak sama");
    }
}

File: Modul 7/NO 3/PRAK603-2210817310016-Riyo Aurora Guison.py
a, b=map(int,input().split())
if(a==b):
    baris1=list(map(int,input().split()))
    baris2=list(map(int,input().split()))
    for i in range (a):
        print(baris1[i]*baris2[i],end=' ')
else:
    print("Jumlah tidak sama")